src/adapters/scene.js

```diff
@@ -399,4 +407,30 @@ Scene.prototype.moveMobileGeolocationButton = function(bottom = 0) {
   }
 };
 
+Scene.prototype.mobileGeolocationButtonVisibility = function(visible) {
+  if (isMobileDevice()) {
```

Comment:
I would advice to use an early return instead, as it is a pre-condition to execute the rest of this function
```suggestion
  if (!isMobileDevice()) {
    return 
  }
```